plotters/src/lib.rs

```diff
@@ -1,4 +1,5 @@
 #![warn(missing_docs)]
+#![cfg_attr(doc_cfg, feature(doc_cfg))]
```

Comment:
This is a requirement at the moment. The ˋdoc_cfgˋ flag is not a feature of the crate but rather a cog flag which is set. Syn, serde, pyo3 and all other crates who use this feature have the exact same statement in their ˋlib.rsˋ file.

Reply:
When uncommenting this line, every
```rust
#[cfg_attr(doc_cfg, doc(cfg(feature = "bitmap_backend")))]
```
generates the following warning:
```bash
error[E0658]: `#[doc(cfg)]` is experimental
   --> plotters/src/lib.rs:917:25
    |
917 |     #[cfg_attr(doc_cfg, doc(cfg(feature = "bitmap_backend")))]
    |                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |
    = note: see issue #43781 <https://github.com/rust-lang/rust/issues/43781> for more information
    = help: add `#![feature(doc_cfg)]` to the crate attributes to enable
    = note: this compiler was built on 2024-01-31; consider upgrading it if it is out of date
```
As far as I understand, the code in question enables the `doc_cfg` feature when the `doc_cfg` flag is passed by the compiler. Theoretically, it should be possible to rename the `doc_cfg` flag used in the compiler invocation
```
cargo +nightly-2024-02-01 rustdoc --all-features -- --cfg some_cfg_flag
```
and accordingly rename the line to
```
#![cfg_attr(some_cfg_flag, feature(doc_cfg))]
```
This is also being done for example [here](https://gist.github.com/jam1garner/7d909e1d0e5986b9b8d50830314bec63).
But by abuse of notation these two distinct concepts are simply named identically since they ultimately serve the same purpose.

Reply:
You're right, I forgot about that. I've used the same code in the past, but I didn't realize that `feature` in this context was about enabling nightly compiler features.